virtcontainers/qemu.go

```diff
@@ -675,9 +671,10 @@ func (q *qemu) setupVirtiofsd() (err error) {
 	}
 
 	err = cmd.Start()
-	if err == nil {
-		q.state.VirtiofsdPid = cmd.Process.Pid
+	if err != nil {
+		return fmt.Errorf("virtiofs daemon %v returned with error: %v", q.config.VirtioFSDaemon, err);
```

Comment:
@c3d, travis is failing here:
```
virtcontainers/qemu.go:675: File is not `gofmt`-ed with `-s` (gofmt)

		return fmt.Errorf("virtiofs daemon %v returned with error: %v", q.config.VirtioFSDaemon, err);

WARN [runner/golint] Golint: can't lint 53 files: no file name for file &{Doc:0xc00dc5ac40 Package:19278537 Name:main Decls:[0xc01f5913c0 0xc01f591400 0xc01f591440 0xc01f591500 0xc01f591540 0xc01f591580 0xc01f5915c0 0xc01f591600 0xc01f591680 0xc01f591700 0xc01f591780 0xc01f591800 0xc01f591840 0xc01f591880] Scope:scope 0xc0256e55e0 {

	var commit

	var version

	var envCmd

	var configFilePathOption

	const name

	const project

	const projectURL

	const defaultRootDirectory

	var showConfigPathsOption

	const projectPrefix

	var checkCmd

	var defaultRuntimeConfiguration

	var defaultSysConfRuntimeConfiguration

}

 Imports:[0xc021557710] Unresolved:[fmt fmt fmt fmt] Comments:[0xc00dc5ac40 0xc00dc5ad80 0xc00dc5ae60 0xc00dc5bdc0 0xc013c3c420 0xc013c3d460 0xc013c3d9e0 0xc013e782c0 0xc013e789a0 0xc013e79b20 0xc013e79f00]} 

The command ".ci/static-checks.sh" failed and exited with 1 during .
```

Seems that you'd need to have the (at least) first %v replaced by %s, so it wouldn't complain.

Reply:
Leftover semicolon. To much coding in C :-)

Reply:
And I didn't even realised that. /o\.

Let me rerun the CI.